src/helm/benchmark/scenarios/fin_qa_scenario.py

```diff
@@ -0,0 +1,75 @@
+import os
+import re
+from typing import List, Any, Dict
+import json
+
+from helm.common.general import ensure_directory_exists, ensure_file_downloaded
+from .scenario import Scenario, Instance, Input, Output, Reference, TRAIN_SPLIT, TEST_SPLIT, CORRECT_TAG
+
+
+_DATASET_URL_PREFIX = "https://github.com/czyssrs/FinQA/raw/0f16e2867befa6840783e58be38c9efb9229d742/dataset/"
+
+
+class FinQAScenario(Scenario):
+    """
+    FinQA is a question answering task over financial reports that requires robust numerical reasoning.
+
+    FinQA: A Dataset of Numerical Reasoning over Financial Data
+    https://arxiv.org/abs/2109.00122
+    https://github.com/czyssrs/FinQA
+
+
+
+    We add the sub-headers "Pre-table text", "Table in JSON format", "Post-table text" to the input. Example:
+
+    ```
+    Pre-table text: printing papers net sales for 2006 decreased 3% ( 3 % ) from both 2005 and 2004 due principally to the sale of the u.s .
+    coated papers business in august 2006 .
+    [more lines]
+
+    Table in JSON format: [["in millions", "2006", "2005", "2004"], ["sales", "$ 6930", "$ 7170", "$ 7135"], ["operating profit", "$ 677", "$ 473", "$ 508"]]
```

Comment:
totally reasonable to do it this way, but i also wonder if this is the best/most natural way? it seems a bit more natural to use markdown formatting? but i dunno